### PR TITLE
Conform to progrium/bashstyle

### PR DESCRIPTION
**Pull request by josegonzalez**
This PR makes sshcommand conform to the style of writing bash that is promulgated in [progrium/bashstyle](https://github.com/progrium/bashstyle).

Most of the code additions are from methods that allow us to reflect the usage of each function.

### CODE REVIEW

File: sshcommand [hunk 1]
```diff
@@ -24,83 +62,129 @@ f_adduser() {
...
-    echo "Usage : sshcommand create user command"
-    echo "        sshcommand acl-add user identifier"
-    echo "        sshcommand acl-remove user identifier"
-    echo "        sshcommand help # shows this usage message"
-    ;;
-
-esac
+log-fail() {
+  declare desc="Log fail formatter"
+  echo "$@" 1>&2
```
Comment (michaelshobbs) [inline sshcommand]: Should `log-fail` exit or return 1 as we do in dokku?

**Comment by michaelshobbs** [comment]
Only briefly looked at this; I can take a look more closely tomorrow. If you **must** ship this today, I'm :+1: as long as tests pass, of course. :smile:

File: sshcommand (hunk 1 above)
Comment (josegonzalez) [inline sshcommand]: Fixed.

**Comment by josegonzalez** [comment]
Shipping it, tests pass and it's more awesomer.

**Pull request merged**